### ISSUE TITLE
Music played after long-clicking a game seems too loud

### ISSUE THREAD

**Issue by HFPasfho** [hrydgard/ppsspp#18865]
### Game or games this happens in

probably all games

### What area of the game / PPSSPP

On the main screen, after holding the game icon, you can open the game screen where the game music plays. This doesn't seem to make sense, but I feel like the music is noticeably louder than the game itself. This is very noticeable in GTA VCS.

### What should happen

I'm not sure if this is a bug, but this music should not be so loud. 

### Platform

Android

### Mobile device model or graphics card (GPU)

Samsung S9

### PPSSPP version affected

v1.17.1

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [ ] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
Yeah, we just play at the max volume here. It might make sense to soften it and/or add a parameter.

**Comment by NABN00B** [comment]
> Yeah, we just play at the max volume here.

This is actually affected by the "Global volume" setting.

**Comment by hrydgard** [comment]
This is now separately configurable with the "UI volume" slider, and it has a lower default. Closing.
